docs/get-started/getting-started.md

```diff
@@ -375,9 +375,97 @@ and `GpuColumnarExchange`.  Those correspond to operations that run on the GPU.
 
 ![Join Example on Spark SQL UI](../img/join-sql-ui-example.png)
 
+## Enabling RapidsShuffleManager
+---
+**NOTE**
+
+The _RapidsShuffleManager_ is a beta feature!
+
+---
+
+The _RapidsShuffleManager_ is an implementation of the `ShuffleManager` interface in Apache Spark,
+that allows custom mechanisms to exchange shuffle data. The _RapidsShuffleManager_ has two components:
+a spillable cache, and a transport that can utilize _Remote Direct Memory Access (RDMA)_ and high-bandwidth 
+transfers within a node that has multiple GPUs. This is possible because the plugin utilizes 
+[Unified Communication X (UCX)](https://www.openucx.org/) as its transport.
+
+- **Spillable cache**: This store keeps GPU data close by where it was produced, and spills to host memory
+or disk when running out of space due to limited GPU memory, or configuration (when going from host
```

Comment:
"or configuration" is oddly worded.  How about describing how it spills to host memory when available GPU memory runs low then separately mention it will spill from host memory to disk when the configurable amount of available host memory for spill runs low.